app/models/article.rb

```diff
@@ -1,6 +1,7 @@
 class Article < ApplicationRecord
 
   has_many :comments, dependent: :destroy
+  belongs_to :user, foreign_key: :user_id
 
-  validates_the_presence_of :title, :author
+  validates_presence_of :title, :author
```

Comment:
Do you think there should also be length validation as regarding the minimum/maximum no of xters

Reply:
There could be, though this PR is majorly for user authentication will incorporate the feedback in the articles PR. Thanks.